pkg/config/middleware_test.go

```diff
@@ -293,41 +290,3 @@ func TestAuthMiddlewareWithUnauthorized(t *testing.T) {
 		}
 	})
 }
-
```

Comment:
Since go 1.12, `monkey.PatchInstanceMethod` is not working anymore, thus remove this hacky monkey patch